### ISSUE TITLE
Clicking the confirm email link twice yields a not-so-friendly error

### ISSUE THREAD

**Issue by kkinder** [plausible/analytics#331]
After signing up, I clicked twice on the confirmation link I got upon joining. Surprisingly, on the second time, I got a JSON dump:

```
[email: {"has already been taken", [constraint: :unique, constraint_name: "users_email_index"]}]
```

I mean, I get it, but that's not a very good UX.

**Comment by ukutaht** [comment]
Good point :) Could be a much better error page.

**Comment by ukutaht** [comment]
Addressed in #331. Thanks for reporting!
